blerry/blerry.be

```diff
@@ -126,6 +133,17 @@ class blerry_helpers
     tasmota.resp_cmnd_done()
   end
 
+  static def cmd_url_config(cmd, idx, url)
+    if string.find(url,'http',0,4)
+      print('BLY: URL does not start with http(s)', url)
+      return false
+    else
```

Comment:
What's going on here?

Reply:
The else is unneeded.  The string.find verifies the url starts with http(s)://.  A very rudimentary validation, not technically needed.